sdk/src/driftClient.ts

```diff
@@ -201,6 +202,8 @@ export class DriftClient {
 			);
 		}
 
+		this.fetchMarketLookupTableAccount();
```

Comment:
dont wanna call async methods in the constructor, can lead to weird race conditions and not everybody will need to use the lookup table